### PR TITLE
Refactor tests, expose apply/generate entries

### PR DESCRIPTION
**Pull request by dy**
This extends the #6 

- exposes apply/generate, very helpful guys
- refactors tests → tape

I would not expose `apply` and `generate` as `index.js` entries - that creates API inconsistency - you can't loop over the all functions.
Instead, i'd go for `./apply` and `./generate` entries.

Although we could export `apply` function by default with window functions defined as properties.

@rreusser @mikolalysenko

### CODE REVIEW

**Comment by dy** [comment]
@rreusser can you please share the access to the npm?

File: README.md [hunk 1]
```diff
@@ -11,33 +11,30 @@ Among other uses, [window functions](http://en.wikipedia.org/wiki/Window_functio
 
 ## Usage
 
-To require all window functions, include the top-level `window-function` module:
+Apply window to a signal:
 
 ```javascript
-var wfuncs = require('window-function')
+var blackmanHarris = require('window-function/blackman-harris')
+var applyWindow = require('window-function/apply')
```
Comment (rreusser) [inline README.md:18]: I like it. 👍

**Comment by rreusser** [comment]
Thanks for the cleanup, @dy! I've added you as a npm maintainer. Publish away! 👏

**Comment by dy** [comment]
Thanks @rreusser!
I guess npm owner has some latency. Will wait for a bit.

**Comment by rreusser** [comment]
It has you listed under maintainers, but not publicly. I think maybe it sent an invitation.

**Comment by dy** [comment]
Yep, done!

**Pull request merged**